### PR TITLE
Separate readreplica chart

### PR DESCRIPTION
**Pull request by ykadowak**
### Description:

This PR separates the read replica related resources into a separate chart and removes them from under helm operator management. The reason for this is that the rotator's process to restart those resources conflicts with the helm operator.

The `values.yaml` of vald-read-replica chart is a symlink to the original `values.yaml` and it'll be copied automatically when packaging by `helm package`.

The deployment procedure for the read replica resources in an operator environment will be as follows:

1. Deploy Vald with helm operator + vrs.yaml
2. Deploy vald-readreplica with helm install + values.yaml (with the same configuration as vrs.yaml)

### Versions:

- Go Version: 1.21.5
- Docker Version: 20.10.8
- Kubernetes Version: v1.28.4
- NGT Version: 2.1.6

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document and completed [our CLA agreement](https://cla-assistant.io/vdaas/vald).
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?

### CODE REVIEW

**Comment by kpango** [comment]
Could you please add README.md?
It would be better to write values.yaml is made by symbolic linked generatrion.

and values.schema.json too

File: charts/vald-readreplica/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Run the following command to install the chart,
+
+    $ helm install vald-readreplica vald/vald-readreplica
+
+## Configuration
+
+### Overview
+
+`values.yaml` is a symbolic link to the `values.yaml` of the main vald chart
```
Comment (kpango) [inline charts/vald-readreplica/README.md:27]: can you add the link to each file ?

File: charts/vald-readreplica/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - artificial-intelligence
+home: https://vald.vdaas.org
+icon: https://raw.githubusercontent.com/vdaas/vald/main/assets/image/svg/symbol.svg
+sources:
+  - https://github.com/vdaas/vald
+maintainers:
+  - name: kpango
+    email: kpango@vdaas.org
+  - name: rinx
+    email: rinx@vdaas.org
```
Comment (vankichi) [inline charts/vald-readreplica/Chart.yaml:44]:
```suggestion
  - name: ykadowak
    email: ykadowak@vdaas.org
```

**Review by vankichi — CHANGES_REQUESTED** [review]

**Review by vankichi — APPROVED** [review]

**Review by kpango — APPROVED** [review]
LGTM

**Review by vankichi — APPROVED** [review]
LGTM

**Pull request merged**